Header header           # Contains timestamp information for the most recent update to the model
WorldObject[] objects   # The list of objects in the world
